travis: Upload build artifacts for release branch.

priyank-p opened the pull request:
Only upload's artifacts for release branch.

priyank-p committed:
travis: Upload build artifacts of PR's.

akashnimare commented on scripts/prepare-artifacts.js:
```diff
@@ -0,0 +1,29 @@
+  file = file.replace(/^z/, 'Z');
+
+  if (/\.deb/.test(file)) {
+    file = file.replace(/[_]{1}/g, '');
```
This should work -
```
file = file.replace(/_/g, '-')
```

akashnimare committed:
Merge branch 'master' into artifacts

priyank-p committed:
artifacts: Upload snap packages too.

priyank-p committed:
apt-packages: Silent the output when installing apt packages.

This reduces noise caused by output when installing apt packages which is
not useful for debugging.

priyank-p committed:
travis: Build x32 bit too.

akashnimare committed:
travis: Add i386 dependency.

akashnimare commented:
@cPhost looks like we can't build the 32-bit installers, right?

priyank-p commented:
Yeah, we can't yet, I will give a try sometime soon but yeah it will have to be with docker images.

akashnimare commented:
@cPhost let's try the docker image.

priyank-p commented:
Finally after some more tries, got something working properly... Opening a new PR.
